### DISCUSSION TITLE
灰原哀是我的！！

### DISCUSSION THREAD

**Discussion by SCJWY** [share121/inter-knot#2288] [其他]
哀殿！！
![微信图片_20240812150754](https://github.com/user-attachments/assets/f32bd0b3-7740-4cca-995f-9fcc28142eff)

**Comment by ywhuuuu** [comment]
不是哥们

**Comment by EustaceSop** [comment]
兄弟轉身有急事

**Comment by duskbirds** [comment]
可爱可爱可爱<img src="https://bbs-static.miyoushe.com/static/2023/04/28/ad3e4d3d37302d240339f611b2458ef9_7921317972787298019.png" alt="米仔_比心" width="75" >
